### ISSUE TITLE
Absolute path in cell 14 (just before Evolution of the Data Mismatch)

### ISSUE THREAD

**Issue by pjuda** [lamd91/mends_testcases#1]
```
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-14-b74814fd5f77> in <module>
      1 iteration = 8 # must be greater than zero
      2 rank_ensMember = 11
----> 3 mends.updatedCoarseVar_multiresoMPS(iteration, rank_ensMember, coarsestGridDim_y, coarsestGridDim_x, finestGridDim_y, finestGridDim_x)

~/mends_testcases/processmends.py in updatedCoarseVar_multiresoMPS(iteration, realizationRank, coarseGridDim_y, coarseGridDim_x, fineGridDim_y, fineGridDim_x)
    379 
    380         beforeIteration = iteration - 1
--> 381         par = np.reshape(np.loadtxt('/home/lamd/Documents/cluster/gwes/results/ens_of_parameters_beforeUpdate_0.txt')[:, realizationRank], (coarseGridDim_y, coarseGridDim_x)) # not shown in figure
    382         parIni_min = np.min(par)
    383         parIni_max = np.max(par)

~/mends_testcases/venv/lib/python3.5/site-packages/numpy/lib/npyio.py in loadtxt(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin, encoding, max_rows)
    979             fname = os_fspath(fname)
    980         if _is_string_like(fname):
--> 981             fh = np.lib._datasource.open(fname, 'rt', encoding=encoding)
    982             fencoding = getattr(fh, 'encoding', 'latin1')
    983             fh = iter(fh)

~/mends_testcases/venv/lib/python3.5/site-packages/numpy/lib/_datasource.py in open(path, mode, destpath, encoding, newline)
    267 
    268     ds = DataSource(destpath)
--> 269     return ds.open(path, mode, encoding=encoding, newline=newline)
    270 
    271 

~/mends_testcases/venv/lib/python3.5/site-packages/numpy/lib/_datasource.py in open(self, path, mode, encoding, newline)
    621                                       encoding=encoding, newline=newline)
    622         else:
--> 623             raise IOError("%s not found." % path)
    624 
    625 

OSError: /home/lamd/Documents/cluster/gwes/results/ens_of_parameters_beforeUpdate_0.txt not found.
```

**Comment by lamd91** [comment]
Fixed by 3ad595bd16c70f36102346f579663c8c62905e0e

**Comment by pjuda** [comment]
Merci, @lamd91 !
